Dockerfile

```diff
@@ -467,309 +452,304 @@ RUN set -x; \
 	&& cd $MW_HOME/extensions/WikiSEO \
-	&& git checkout -q 610cffa3345333b53d4dda7b55b2012fbfcee9de \
+	&& git checkout -q 5d27adb6ec53e7b1076a485e93f0b0c272d08937 \
 	# WSOAuth
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/WSOAuth $MW_HOME/extensions/WSOAuth \
 	&& cd $MW_HOME/extensions/WSOAuth \
-	&& git checkout -q 3c54c4899dd63989bc3214273bf1c5807c7ac5db
+	&& git checkout -q 9f25fade2bcd3fae44c3236072f5ea2eb068b75c
 
 #### WikiTeq extensions ####
 
+# A
+RUN set -x; \
+	cd $MW_HOME/extensions \
+  	# AddMessages
+	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/AddMessages $MW_HOME/extensions/AddMessages \
+	&& cd $MW_HOME/extensions/AddMessages \
+	&& git checkout -q a0af32f229d93016f3c3e80bcf2065e09f498064 \
+	# Auth_remoteuser
+	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/Auth_remoteuser $MW_HOME/extensions/Auth_remoteuser \
+	&& cd $MW_HOME/extensions/Auth_remoteuser \
+	&& git checkout -q c985d520c7aea38092ee7208be31f07a7251210d
+
 # B
 RUN set -x; \
 	cd $MW_HOME/extensions \
 	# Buggy
 	&& git clone --single-branch -b $MW_VERSION https://github.com/wikimedia/mediawiki-extensions-Buggy.git $MW_HOME/extensions/Buggy \
 	&& cd $MW_HOME/extensions/Buggy \
-	&& git checkout -q 768d2ec62de692ab62fc0c9f1820e22058d09d4b
+	&& git checkout -q 49dc8fd9ae01195127e0106b94fb01beb5022eb7
 
 # C
 RUN set -x; \
 	cd $MW_HOME/extensions \
   	# ChangeAuthor
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/ChangeAuthor $MW_HOME/extensions/ChangeAuthor \
 	&& cd $MW_HOME/extensions/ChangeAuthor \
-	&& git checkout -q c297a88407c6dea60dfa03b1a7d5f4cd78d9e0c9 \
+	&& git checkout -q 89105aebc39d85c15149391832376d6ea85702c1 \
 	# Citoid
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/Citoid $MW_HOME/extensions/Citoid \
 	&& cd $MW_HOME/extensions/Citoid \
-	&& git checkout -q 1e605c7d89368c334cbe83b4da8e1b6d72ae9c33
-
-# D
-RUN set -x; \
-	cd $MW_HOME/extensions \
-	# DiscussionTools
-	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/DiscussionTools $MW_HOME/extensions/DiscussionTools \
-	&& cd $MW_HOME/extensions/DiscussionTools \
-	&& git checkout -q c667c3c020a61010a7598438a8122f015678b242
+	&& git checkout -q ee1f920556ff245fd4366d8ceed15afb6aa1b0f8
 
 # E
 RUN set -x; \
 	cd $MW_HOME/extensions \
    	# EditAccount
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/EditAccount $MW_HOME/extensions/EditAccount \
 	&& cd $MW_HOME/extensions/EditAccount \
-	&& git checkout -q a9a0d7002483d2165300afd108a8f4b3c2c485ff \
+	&& git checkout -q e59561c56836a98bc5f5f3068bc81fd67a5d1c41 \
 	# Flow
```

Comment:
We really should wipe the Flow from the image soon

Reply:
yeah, but it'll probably be easier to migrate any uses on a 1.43 wiki than a 1.39 wiki